src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+}
+
+.stars :not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars-star {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+}
+
+.stars--4 :nth-child(-n+4) {
```

Comment:
you should add all conditions for different amount of stars here (just reuse code from stars task)